index.qmd

```diff
@@ -19,18 +30,18 @@ If you have a question or idea for this guide, please start a [Github Discussion
 
 ## The opportunity
 
-Just putting data on the cloud does not solve the big geospatial data problem. Users cannot reasonably wait to download, store and work with large files on their machines. To have access to data in memory, large volumes of data must be available via subsetting methods.
+Storing data in the cloud does not on its own solve geospatial's data problem. Users cannot reasonably wait to download, store, and work with large files on their machines. Large volumes of data must be available via subsetting methods to access data in memory.
```

Comment:
I prefer the first sentence as is, although I'm willing to be overruled @wildintellect @kylebarron

Reply:
I will wait for @wildintellect and @kylebarron feedback. I prefer the latter, personally, more in terms of the verb selection.

Reply:
I don't have a strong preference. I see `just putting` and `on its own solve` as somewhat similar meanings.